### PR TITLE
Seamless switching

### PR DESCRIPTION
**Pull request by kuehnhammer**
This is the cleaned up baseline for seamless switching, as demonstrated in the last developer call.

### CODE REVIEW

File: app/controllers/api/middleware.js [hunk 1]
```diff
@@ -33,3 +33,6 @@ exports.files = async(function*(req, res) {
 exports.services = async(function*(req, res) {
   proxy_call(req, res, "/mw-api/services");
 });
+exports.service_announcement = async(function*(req, res) {
+  proxy_call(req, res, "/mw-api/service_announcement");
```
Comment (dsilhavy) [inline app/controllers/api/middleware.js:37]: Is this route used somewhere at the moment? In `js/middleware.js` I only see calls to `/api/mw/services` and `/api/mw/files`

File: package.json [hunk 2]
```diff
@@ -4,19 +4,22 @@
   "description": "Shared web interface for the 5G-MAG processes: modem and middleware",
   "main": "app.js",
```
Comment (dsilhavy) [inline package.json:5]: Please increase version number to 0.1.0

**Review by dsilhavy — CHANGES_REQUESTED** [review]

File: app/controllers/api/middleware.js (hunk 1 above)
Comment (kuehnhammer) [reply app/controllers/api/middleware.js:37]: It is, there was some code missing in app/assets/js/middleware.js

**Comment by kuehnhammer** [comment]
@dsilhavy  - OK, I added the requested changes,

**Comment by dsilhavy** [comment]
The changes from this PR are part of #9 , closing this one

**Pull request closed without merging**